moochub-schema.json

```diff
@@ -1,555 +1,3105 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
-  "$id": "https://github.com/moochub/schema/releases/tag/v2.3",
+  "$id": "https://github.com/moochub/schema/releases/tag/v3.0",
   "title": "JSON schema for MOOC providers integrated into moochub.org",
   "description": "This schema specifies the JSON format each MOOC provider must offer in order to be integrated into moochub.org",
   "type": "object",
   "properties": {
-    "links": {
-      "title": "Links",
+    "id": {
+      "type": "string",
+      "description": "The ID of the course as an UUID.",
+      "minLength": 1,
```

Comment:
If it's a UUID, it has a specified format.

Reply:
I just checked some of our partners e.g. iMoox: they use simple numbers 1, 2, 3 ... as incremental values for all listed courses. Should we enforce UUID? I wrote an email to Beni from iMoox to clarify.

Reply:
is now UUID, also by datatype